src/frontend/src/lib/components/convert/ConvertValue.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	export let ref: undefined | string = undefined;
+</script>
+
+<div class="mb-2 flex w-full justify-between last:mb-0 md:items-center">
+	<label for={ref} class="mr-1 text-sm sm:mr-2 sm:text-base"><slot name="label" /></label>
+
+	<div class="flex flex-col items-end sm:flex-row sm:items-center">
+		<span
+			class={`mb-1 text-sm font-bold sm:mb-0 sm:text-base ${$$slots['secondary-value'] ? 'mr-0 sm:mr-3' : ''}`}
+			><slot name="main-value" /></span
+		>
+
+		<span class="text-sm text-aurometalsaurus"><slot name="secondary-value" /></span>
```

Comment:
I would add the same TODO you used in PR #3452 , just to keep track of the colors

(comment https://github.com/dfinity/oisy-wallet/pull/3452/files#r1838102292)

Reply:
Done!